packages/mds-policy-service/test_data/policies.ts

```diff
@@ -1,4 +1,18 @@
-import { Policy } from '@mds-core/mds-types'
+/**
+ * Copyright 2021 russenguggemos
```

Comment:
Add the following property to `vscode/settings.json` in this repo and change here.
```json
"licenser.author": "City of Los Angeles"
```

Reply:
thanks!